models/topics-models.js

```diff
@@ -1,8 +1,10 @@
 const db = require("../db/connection");
 function selectTopics() {
   return db.query("SELECT * FROM topics").then((results) => {
+    if (results.rows.length === 0) {
```

Comment:
What's not found in this instance? We definitely have a topics table. It could indeed be empty but, even if it was, the resource would still be found. I think in the instance of this endpoint it's fine to respond with an array (empty or not) and 200.